src/json/constant.json

```diff
@@ -560,7 +561,7 @@
 		"id": "Bahasa Indonesia",
 		"it": "Italiano",
 		"it-IT": "Italiano (Italia)",
-		"ko": "한국어",
```

Comment:
Please revert this change, it's responsible for spellchecker and not for interface language.